packages/rocketchat-smarsh-connector/server/settings.js

```diff
@@ -0,0 +1,41 @@
+RocketChat.settings.addGroup('Smarsh', function addSettings() {
+	this.add('Smarsh_Enabled', false, {
+		type: 'boolean',
+		i18nLabel: 'Smarsh_Enabled',
+		enableQuery: {
+			_id: 'From_Email',
+			value: {
+				$exists: 1,
+				$ne: ''
+			}
+		}
+	});
+	this.add('Smarsh_Email', '', {
+		type: 'string',
+		i18nLabel: 'Smarsh_Email',
+		placeholder: 'email@domain.com'
+	});
+	this.add('Smarsh_Interval', 'every_30_minutes', {
+		type: 'select',
+		values: [{
+			key: 'every_30_seconds',
+			i18nLabel: 'every_30_seconds'
```

Comment:
This 30 seconds one is more for testing purposes and not really for meant for production (so we don't have to wait a while), so before we release this to the `master` branch we will want to remove this option.